private-distributed.yml

```diff
@@ -5,6 +5,7 @@ allow-licenses:
   - 'Apache-2.0 AND BSD-3-Clause AND Python-2.0'
   - 'Beerware'
   - 'BlueOak-1.0.0'
+  - 'BSD'
```

Comment:
From what I see, BSD is not a valid SPXD identifier: https://spdx.org/licenses/

So, can you ask the project owner to use a valid license?

Reply:
Oh! Yep I'll open an issue in their repository.

Closing this PR, thanks!

Reply:
@JPLachance I added a comment in an already existing issue in the project:

https://github.com/maleck13/readline/issues/26

However, the issue was created in 2019 and hasn't been acknowledged by the project owner.

Also there hasn't been any commits in this project since 8 years.

This is not even a direct dependency: it's a subdependency of things like @types/lodash and core-js...

Do we really have to validate licenses beyond our direct dependencies?...

Reply:
I'll go further: should we even validate licenses in dev dependencies?